src/rgw/rgw_lua_utils.h

```diff
@@ -6,7 +6,6 @@
 #include <ctime>
 #include <lua.hpp>
 
-class CephContext;
```

Comment:
this header file references the `CephContext`, while it fails to include the declaration of `CephContext`. so we'd better do so one way or another.

i'd suggest `#include "include/common_fwd.h"` instead.

Reply:
> i'd suggest `#include "include/common_fwd.h"` instead.

Added `#include "include/common_fwd.h` looks better now, 
tested that still compiles OK with both clang and gcc after the include.